### ISSUE TITLE
Emergency!  explicitBatch can not work,  Help!!!!

### ISSUE THREAD

**Issue by zihao-lu** [NVIDIA/TensorRT#1492]
## Description
I have been dealing with this problem for two day....
I use trtexec to generate a tensorRT engine with  "./trtexec --onnx=/home/lzh/resnet50.onnx --saveEngine=resnet50.trt ", 
resnet50.onnx was exported using resnet-50 model importing from torchvision, like this:

    import torchvision.models as models
    import torch
    import torch.onnx
  
    model = models.resnet50(pretrained=True)
    model.eval()
    dummy_input = torch.randn((16, 3, 224, 224))
    torch.onnx.export(model, dummy_input, "resnet50.onnx", verbose=True)

**Then, i run the command above to generate tensorrt engine, it's ok, i got right result.**

**BUT,
when i run "./trtexec --onnx=/home/lzh/resnet50.onnx --saveEngine=resnet50.trt --explicitBatch", i got erro like this:**

  [08/15/2021-21:34:38] [E] [TRT] (Unnamed Layer* 0) [Convolution]: at least 4 dimensions are required for input
  [08/15/2021-21:34:38] [I] [TRT] 321:Conv ->
  While parsing node number 1 [BatchNormalization -> "322"]:
  3
  --- Begin node ---
  input: "321"
  input: "bn1.weight"
  input: "bn1.bias"
  input: "bn1.running_mean"
  input: "bn1.running_var"
  output: "322"
  op_type: "BatchNormalization"
  attribute {
  name: "epsilon"
  f: 1e-05
  type: FLOAT
   }
  attribute {
  name: "momentum"
  f: 0.9
  type: FLOAT
   }
                
  --- End node ---
  ERROR: /home/lzh/onnx-tensorrt/builtin_op_importers.cpp:661 In function importBatchNormalization:
  [6] Assertion failed: scale_weights.shape == weights_shape
  [08/15/2021-21:34:38] [E] Failed to parse onnx file
  [08/15/2021-21:34:38] [E] Parsing model failed
  [08/15/2021-21:34:38] [E] Engine could not be created
  &&&& FAILED TensorRT.trtexec # ./trtexec --onnx=/home/lzh/resnet50.onnx --saveEngine=resnet50.trt --workspace=10240 --explicitBatch

I can't deal with it, NEED HELP! THX

## Environment

**TensorRT Version**: 6.0
**NVIDIA GPU**: v100
**NVIDIA Driver Version**:  470.57.02
**CUDA Version**: 10.0
**CUDNN Version**: 7.6.5
**Operating System**: ubuntu 16.0
**Python Version (if applicable)**: 3.6.10
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 1.2
**Baremetal or Container (if so, version)**:

**Comment by ttyio** [comment]
Hi @zihao-lu ,could you upgrade to latest TRT8.0? thanks!

**Comment by ttyio** [comment]
close since no activity for more than 3 weeks, please reopen if you still have question, thanks!
